go.mod

```diff
@@ -122,7 +122,7 @@ require (
 	github.com/mitchellh/reflectwalk v1.0.1 // indirect
 	github.com/moby/spdystream v0.2.0 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v1.0.1 // indirect
+	github.com/modern-go/reflect2 v1.0.2 // indirect
```

Comment:
Please don't modify the `go.mod` file unless it's absolutely necessary at this point. My pending "multi-port-support" PR contains several changes and upgrades (including this one), so this might cause merge conflicts.

Reply:
There is a panic while serializing a `core.Secret` struct that happens on `github.com/json-iterator/go v1.1.10` that is resolved in `v1.1.12`.

Reply:
I had no idea that this PR would fix problems related to serializing a `core.Secret`. Is that a linux specific problem?